boards/arm/arduino_due/arduino_due.dts

```diff
@@ -15,6 +15,7 @@
 	chosen {
 		zephyr,sram = &sram0;
 		zephyr,flash = &flash0;
+		zephyr,code-partition = &boot_partition;
```

Comment:
You don't want this setting if you want Zephyr to be chain-loaded by mcuboot. Why add it here? The default behavior should be that Zephyr can use all of flash.